specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/CommonDefinitions.json

```diff
@@ -34,6 +34,19 @@
         }
       }
     },
+    "WorkloadProfileType": {
+      "description": "Workload profile type for container apps to execute.",
+      "enum": [
+        "General Purpose",
+        "Memory Optimized",
+        "Compute Optimized"
```

Comment:
>"Compute Optimized [](http://example.com/codeflow?start=8&length=18)

Is it ok to have spaces in these enums? Maybe it is - we just never had that before in this swagger (or in app service)